pyne/dbgen/hashtools.py

```diff
@@ -12,7 +12,7 @@
 QA_warn(__name__)
 
 # list of nodes from distinct data sets
-nodelist = ['/atomic_mass', '/material_library',
+nodelist = ['/atomic_mass', '/material_library/materials',
```

Comment:
Confused here. In other locations the path is updated from `/material_library` to `/materials`, but here `materials` is appended to the original path?

Reply:
because for a material library the name of the groups is nested inside the `material_library` dataset

Reply:
Ah, I see -- the base is always `/material_library`

https://github.com/pyne/pyne/blob/develop/src/material_library.cpp#L245

Sorry, it's been a while since I've looked at this.

Reply:
it is kinda new... been working on it for a year been merged a month or 2 ago...